app/backend/src/tests/test_fixtures.py

```diff
@@ -68,37 +68,28 @@ def create_schema_from_models():
     Base.metadata.create_all(get_engine())
 
 
-def db_impl(param):
+def db_impl():
     """
-    Connect to a running Postgres database
-
-    param tells whether the db should be built from alembic migrations or using metadata.create_all()
+    Connect to a running Postgres database, build it using metadata.create_all()
     """
 
     # running in non-UTC catches some timezone errors
-    # os.environ["TZ"] = "Etc/UTC"
     os.environ["TZ"] = "America/New_York"
 
     # drop everything currently in the database
     drop_all()
 
-    if param == "migrations":
-        # rebuild it with alembic migrations
-        apply_migrations()
-    else:
-        # create everything from the current models, not incrementally through migrations
-        create_schema_from_models()
+    # create everything from the current models, not incrementally through migrations
+    create_schema_from_models()
 
 
-@pytest.fixture(params=["migrations", "models"])
-def db(request):
+@pytest.fixture()
+def db():
```

Comment:
I think this function is now unneeded, it was used somewhere in the communities tests, but I don't think this is required anymore?

I might be wrong though

Reply:
db_impl is called from communities test, but I'm leaning towards not using a fixture but a simple Python function `recreate_database()` that all tests call in the beginning of the test function. What do you think? Since there's no teardown code we don't win anything by using a fixture.

Reply:
Why not use fixtures?

Reply:
They are unnecessary magical. Nothing beats pure Python code when checking what the tests does.

Reply:
Yeah fair enough. As long as we feel we won't need that magic, then why not?

Reply:
We might want to add magic in the future, so let's keep it as a fixture for now :)